initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -39,6 +39,46 @@ message() {
 MESSAGE_MARKER
 }
 
+adjustdir() {
+    local dir=${1}
```

Comment:
A comment might be useful (I think this is canonicalizing the incoming directory so it doesn't have a trailing /).